### ISSUE TITLE
question for video retrieval

### ISSUE THREAD

**Issue by yangjax** [willard-yuan/CNN-for-Image-Retrieval#12]
hi, i am very interested in your work on image retrieval, and my question is how can i apply it to the video retrieval  domains.  there mainly two pints make me confused:
firstly, image retrieval take CNN as the feature extractor , and the CNN mainly trained with the classification loss on such cifar-10 or landmark dataset(Oxf5k), can it generalize to the actual video scenarios(usually are not landmark)?  fine-tuning needed? and how?
secondly, how can i calculate the similarity between the query video and reference video, my method is dividing video into key frames and match the similar frames by extracting the frame features
 can you give me some opinion. thanks!

**Comment by willard-yuan** [comment]
@yangjax Sorry for later reply. For video retrieval, I recommend you use frame-based method since video-based feature extraction is very time-consuming, and it's unacceptable for large-scale video dataset. The pre-trained imageNet feature extraction model is a good start, and you can use PQ method as the ANN index. To calculate the similarity between the query video and reference video, [the post](http://yongyuan.name/blog/multi-frames-ranking-problem.html) I wrote is very useful and it's very effective in real world application.

Thank you for your interest for my work.
